tidb-cloud/tidb-cloud-console-auditing.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Console Audit Logging
+Summary: Learn about how to audit operations on the TiDB Cloud Console.
+---
+
+# Console Audit Logging
+
+TiDB Cloud provides you with the audit log feature for Console operations, which can record the history of user access details (such as user login to the Console and cluster creations).
+
+> **Note:**
+>
+> - Currently, the **audit logging** feature is experimental. The interface and output are subject to change.
+> - Console audit logging is enabled by default and the log is not accessible to users without the help of PingCAP support.
+
+To assess whether user access policies and other information security measures of your organization are effective or not, it is a security best practice to periodically analyze the console audit logs.
+
+## Specify auditing filter rules
+
+The console audit logging feature is enabled by default, and all user operations on the Console will be recorded in the audit logs.
+Audit logs cover the following event types:
+
+| Audit Event Type | Description |
+|---|---|
+| AuditEventSignIn | sign in |
+| AuditEventSignOut | sign out |
+| AuditEventUpdateUserProfile | update user's first name and last name |
+| AuditEventUpdateMFA | enable or disable MFA |
+| AuditEventCreateProject | create a new project |
+| AuditEventUpdateProject | update the project name |
+| AuditEventDeleteProject | delete a project |
+| AuditEventInviteUserIntoProject | invite a user into project |
+| AuditEventDeleteProjectUser | delete a project user |
+| AuditEventUpdateOrg | update organization name and time zone |
+| AuditEventCreateIntegration | create an integration |
+| AuditEventDeleteIntegration | delete an integration |
+| AuditEventListOrgUsers | list users in organization |
+| AuditEventListProjectUsers | list users in a project |
+| AuditEventAddNewPaymentMethod | add a new credit card |
+| AuditEventUpdatePaymentMethod | update credit card info |
+| AuditEventDeletePaymentMethod | delete a credit card |
+| AuditEventCreateAWSVpcPeering | create an aws vpc peering |
+| AuditEventCreateGCPVpcPeering | create a gcp vpc peering | 
+| AuditEventListAWSVpcPeering | list all aws vpc peering in the project |
```

Comment:
These words should be capitalized:

- aws -> AWS
- gcp -> GCP
- vpc peering -> VPC Peering
- The first letter of the first word: Delete, List, Add, Update, Create, Get, etc.